amd_openvx/openvx/include/VX/vx_compatibility.h

```diff
@@ -46,8 +46,10 @@
 #define VX_CONTEXT_ATTRIBUTE_EXTENSIONS                         VX_CONTEXT_EXTENSIONS
 #define VX_CONTEXT_ATTRIBUTE_CONVOLUTION_MAXIMUM_DIMENSION      VX_CONTEXT_CONVOLUTION_MAX_DIMENSION
 #define VX_CONTEXT_ATTRIBUTE_OPTICAL_FLOW_WINDOW_MAXIMUM_DIMENSION      VX_CONTEXT_OPTICAL_FLOW_MAX_WINDOW_DIMENSION
-#define VX_CONTEXT_ATTRIBUTE_IMMEDIATE_BORDER_MODE                      VX_CONTEXT_IMMEDIATE_BORDER
```

Comment:
We should not edit the header files, they come from Khronos.

https://www.khronos.org/registry/OpenVX/

Reply:
@kiritigowda The new version asks to query context for 3-4 new things that were not in the previous versions. They are not included in the header file. How do I go about this?

Reply:
We can add this to vx_ext_amd.h